src/System.Net.Security/tests/FunctionalTests/ClientAsyncAuthenticateTest.cs

```diff
@@ -27,20 +27,14 @@ public class ClientAsyncAuthenticateTest
 
         private static readonly SslProtocols[] s_eachSslProtocol = new SslProtocols[]
         {
-            SslProtocols.Ssl2,
```

Comment:
We're still thinking if Ssl2 should exist in the API surface. Until we decide to remove it, I think the Windows tests should still test Ssl2. 

Wouldn't this removal mean that the Windows tests won't exercise Ssl2?